content/applications/websites/ecommerce/taxes.rst

```diff
@@ -1,13 +0,0 @@
-:nosearch:
-
-=============
-Collect taxes
-=============
-
-.. toctree::
-   :titlesonly:
-
-   ../../finance/accounting/taxation/taxes/default_taxes
-   ../../finance/accounting/taxation/taxes/fiscal_positions
-   ../../finance/accounting/taxation/taxes/taxes
-   ../../finance/accounting/taxation/taxes/taxcloud
```

Comment:
@odoo/ecommerce-doc-review 

I removed this double indexation from e-commerce. I know @toaa-odoo did the same in 16.0... Ideally, if (when) this goes through, we'd need something to replace this. The best is to add a written paragraph or see also section somewhere else to inform users to check the taxes docs.